### PR TITLE
chore: add nft support

### PR DESCRIPTION
**Pull request by kaladinlight**
- update blockbook client types
- use new values from blockbook to return nft ids and values from unchained api

### CODE REVIEW

File: node/coinstacks/common/api/src/evm/service.ts [hunk 1]
```diff
@@ -357,16 +357,45 @@ export class Service implements Omit<BaseAPI, 'getInfo'>, API {
...
-      })),
+      tokenTransfers: tx.tokenTransfers?.map((tt) => {
+        const value = (() => {
+          switch (tt.type) {
+            case 'ERC721':
+            case 'BEP721':
+              return '1'
+            case 'ERC1155':
+            case 'BEP1155':
+              return tt.multiTokenValues?.[0].value ?? '0'
```
Comment (kaladinlight) [inline node/coinstacks/common/api/src/evm/service.ts:368]: multi tokens can be multiple of the same id

File: node/coinstacks/common/api/src/evm/service.ts [hunk 2]
```diff
@@ -357,16 +357,45 @@ export class Service implements Omit<BaseAPI, 'getInfo'>, API {
...
+            default:
+              return tt.value
+          }
+        })()
+
+        const id = (() => {
+          switch (tt.type) {
+            case 'ERC721':
+            case 'BEP721':
+              return tt.value
```
Comment (kaladinlight) [inline node/coinstacks/common/api/src/evm/service.ts:378]: blockbook returns the nft id under `value`, use as `id`

File: node/coinstacks/common/api/src/evm/service.ts [hunk 3]
```diff
@@ -357,16 +357,45 @@ export class Service implements Omit<BaseAPI, 'getInfo'>, API {
...
+            case 'ERC1155':
+            case 'BEP1155':
+              return tt.multiTokenValues?.[0].id
```
Comment (kaladinlight) [inline node/coinstacks/common/api/src/evm/service.ts:381]: get multi token id (not sure why it was set up as an array on the blockbook side, only ever length of 1 at this time)

File: node/coinstacks/common/api/src/evm/service.ts (hunk 1 above)
Comment (gomesalexandre) [reply node/coinstacks/common/api/src/evm/service.ts:368]: 0 looks correct here, though do we want to guard against undefined `multiTokenValues[0]` as a paranoia check?
https://github.com/trezor/trezor-suite/blob/ddd24ef647b9d088c26c0d337cd2de3c9d4b1d56/suite-common/wallet-utils/src/transactionUtils.ts#L413-L416

File: node/coinstacks/common/api/src/evm/service.ts (hunk 3 above)
Comment (gomesalexandre) [reply node/coinstacks/common/api/src/evm/service.ts:381]: Ditto, referring to trezor-suite as the source of truth 0 looks correct, though we might want to guard against `multiTokenValues[0]` being defined, or alternatively, `multiTokenValues?.length`

**Review by gomesalexandre — COMMENTED** [review]
Not an unchained wizard but cross-referencing the blockbook data consumption in trezor-suite, LGTM

File: node/coinstacks/common/api/src/evm/service.ts (hunk 1 above)
Comment (kaladinlight) [reply node/coinstacks/common/api/src/evm/service.ts:368]: they are just checking length and then grabbing id, I think we are good unless we see the api change

Comment (kaladinlight) [reply node/coinstacks/common/api/src/evm/service.ts:368]: meh, doesn't hurt, adding lol

File: node/coinstacks/common/api/src/evm/service.ts (hunk 3 above)
Comment (kaladinlight) [reply node/coinstacks/common/api/src/evm/service.ts]: doin it

**Review by gomesalexandre — APPROVED** [review]
Conceptual stamp

**Pull request merged**